admin-dev/themes/new-theme/js/pages/order/order-shipping-manager.js

```diff
@@ -27,11 +27,7 @@ import OrderViewPageMap from './OrderViewPageMap';
 const $ = window.$;
 
 export default class OrderShippingManager {
-  constructor() {
-    this._initOrderShippingUpdateEventHandler();
-  }
-
-  _initOrderShippingUpdateEventHandler() {
+  initOrderShippingUpdateEventHandler() {
```

Comment:
This could be performed in a constructor only once by using deferred events from jQuery, but it's just a suggestion

```suggestion
  $(OrderViewPageMap.mainDiv).on('click', OrderViewPageMap.showOrderShippingUpdateModalBtn, (event) => {
```

This allows not to init the event each time the DOM is updated